README.md

```diff
@@ -44,6 +44,8 @@ Help of any kind is highly appreciated. Want to pitch in but not sure how?
 - Have Twitter followers? [Tweet about the project!](http://twitter.com)
 - Revise the [cover letter](https://github.com/Benjamin-Lee/deep-rules/edit/master/content/presubmission_inquiry.md)
 
+For more information, see [the contribution guidelines](CONTRIBUTING.md). All contributions are subject to the [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
What you have so far looks good.  I expect you'll add `CODE_OF_CONDUCT.md` before merging.

Double check that `CONTRIBUTING.md` in the link here matches the filename, currently `contributors.md`.

Reply:
CODE_OF_CONDUCT.md is already live -- merged that without a PR. CONTRIBUTING.md is the contribution guidelines, whereas contributors.md is the list of contributors. 

Is that unclear?

Reply:
Yes, this is clear.  I didn't realize what files had been added in 43bc6ee5b9c804b1884226ff22e4e8d415abdd89